google/cloud/firestore_v1/document.py

```diff
@@ -500,7 +500,8 @@ def on_snapshot(self, callback):
         provided callback is run on the snapshot.
 
         Args:
-            callback(Callable[[:class:`~google.cloud.firestore.document.DocumentSnapshot`], NoneType]):
+            callback(Callable[List[:class:`~google.cloud.firestore_v1.document.DocumentSnapshot`], \
```

Comment:
Space should be added after first `callback` word

Reply:
Should i add the space, because i didn't change that thing, i have only added that parameters